client/landing/jetpack-cloud/sections/scan/main.tsx

```diff
@@ -102,18 +142,35 @@ class ScanPage extends Component {
 			return <div className="scan__is-loading" />;
 		}
 
-		const status = scanState.state;
-		if ( status === 'scanning' ) {
+		const { state, mostRecent, threats } = scanState;
+
+		// @todo: find out what should we do when the scan state is 'provisioning' (design missing)
+		// @todo: figure out if these states can happen at all
+		if ( state === 'unavailable' || state === 'provisioning' ) {
```

Comment:
Provisioning is an rare stare. I think we just need to tell the user to retry and we can try fetching the /scan endpoint every 15 seconds or so.

Reply:
Sounds good to me!

I need to say that I just saw this happening. I created an ephemeral site, purchased Backup and Scan, and then Jetpack Cloud Scan was in the `provisioning` state.